### PR TITLE
[distributions] use cholesky_inverse to compute precision matrix

### PR DESCRIPTION
**Pull request by fehiepsi**
Resolves a long-standing TODO. :D 

I also fix the docs of lowrank_mvn which is raised at [forum](https://discuss.pytorch.org/t/lowrankmultivariatenormal-example-raises-valueerror/65381).

cc @vishwakftw

### CODE REVIEW

File: torch/distributions/multivariate_normal.py [hunk 1]
```diff
@@ -182,9 +182,7 @@ def covariance_matrix(self):
 
     @lazy_property
     def precision_matrix(self):
-        # TODO: use `torch.potri` on `scale_tril` once a backwards pass is implemented.
-        scale_tril_inv = torch.inverse(self._unbroadcasted_scale_tril)
-        return torch.matmul(scale_tril_inv.transpose(-1, -2), scale_tril_inv).expand(
+        return torch.cholesky_inverse(self._unbroadcasted_scale_tril).expand(
```
Comment (vishwakftw) [inline torch/distributions/multivariate_normal.py]: We still don’t have batching for cholesky_inverse (it’s in progress)

Comment (fehiepsi) [reply torch/distributions/multivariate_normal.py]: Ah, my mistake! I should use cholesky_solve here...

**Comment by vishwakftw** [comment]
Any performance effects?

**Review by vishwakftw — APPROVED** [review]
Awesome! Thank you for taking care of this.

**Pull request closed without merging**